data_import/views.py

```diff
@@ -26,8 +25,11 @@ def get_and_log(self, request, key_object=None):
         """
         user = request.user if request.user.is_authenticated else None
 
-        if key_object:
-            key = {
+        access_log = NewDataFileAccessLog(
+            user=user, ip_address=get_ip(request), data_file=self.data_file
+        )
+        access_log.data_file_key = json.dumps(
```

Comment:
Is this storing a string rather than a dict in the JSONField?

Reply:
Oh!
Let me double check that.